apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -49,9 +49,16 @@ defmodule OMG.Watcher.DB.Transaction do
     belongs_to(:block, DB.Block, foreign_key: :blknum, references: :blknum, type: :integer)
   end
 
-  def get(hash) do
-    __MODULE__
-    |> Repo.get(hash)
+  def get(hash, preload_block \\ false) do
+    transaction =
+      __MODULE__
+      |> Repo.get(hash)
```

Comment:
Let's load block with tx - it's not much
```suggestion
      |> Repo.get(hash)
      |> Repo.preload(:block)
```